doc/source/tutorial/stats/continuous_arcsine.rst

```diff
@@ -34,4 +34,11 @@ Defined over :math:`x\in\left[0,1\right]`.  To get the JKB definition put :math:
 
      l_{\mathbf{x}}\left(\cdot\right)=N\log\pi+\frac{N}{2}\overline{\log\mathbf{x}}+\frac{N}{2}\overline{\log\left(1-\mathbf{x}\right)}
 
+References
+----------
+
+.. [1] Norman Johnson, Samuel Kotz, and N. Balakrishnan, Continuous Univariate Distributions, second edition, Volumes I and II, Wiley & Sons, 1994.
+
+.. [2] "Arcsine Distribution", Wikipedia, https://en.wikipedia.org/wiki/Student%27s_t-distribution
```

Comment:
The link should be https://en.wikipedia.org/wiki/Arcsine_distribution

Reply:
Oops! Copy paste gone wrong. Thanks!